opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-csv-domain.ts

```diff
@@ -93,6 +93,20 @@ export const ingestionCsvEditField = async (context: AuthContext, user: AuthUser
   return notify(BUS_TOPICS[ABSTRACT_INTERNAL_OBJECT].EDIT_TOPIC, element, user);
 };
 
+export const ingestionCsvResetState = async (context: AuthContext, user: AuthUser, ingestionId: string) => {
+  await patchCsvIngestion(context, user, ingestionId, { current_state_hash: '' });
+  const ingestionUpdated = await findById(context, user, ingestionId);
```

Comment:
You can probably save a call to elastic by using the return value of patchCsvIngestion instead of calling additional findById

Reply:
As far as I remeber I added findById in purpose because the patch is not returning what I need.